spec/models/quote_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Quote, type: :model do
+  let!(:quote) {FactoryBot.create(:quote)}
```

Comment:
Also please create a test for the factory, where you generate 3 quotes and make sure their `:order` value is sequenced properly (1, 2, 3)